packages/apps/test/initSettings.spec.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017-2021 @polkadot/apps authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { validateURL } from '../src/initSettings';
+
+describe('Test Init Function', () => {
+  test('should return URL for ws://localhost:9944', (done) => {
+    const result = validateURL('ws://localhost:9944');
+
+    expect(result).toBeTruthy();
+    done();
+  });
+
+  test('should return URL for ws://127.0.0.1:9944', (done) => {
+    const result = validateURL('ws://127.0.0.1:9944');
+
+    expect(result).toBeTruthy();
+    done();
+  });
+
+  test('should throw error for ws://localhost:65000', (done) => {
+    expect(() => {
+      validateURL('ws://localhost:65000');
+    }).toThrowError('Invalid ws port');
```

Comment:
Group all tests which return `Invalid we port` and use `jest.each` to run them with 1 test.